packages/experimental/Divider/src/DividerTokens.ts

```diff
@@ -0,0 +1,49 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { DividerTokens } from './Divider.types';
+
+export const defaultDividerTokens: TokenSettings<DividerTokens, Theme> = (theme: Theme) =>
+  ({
+    // base tokens
+    contentPadding: 0,
+    flexAfter: 1,
+    flexBefore: 1,
+    insetSize: 0,
+    minWidth: 0,
+    minHeight: 0,
+    thickness: 1,
+    // for aligning the divider if alignContent !== 'center'
+    alignStart: {
+      flexBefore: 0,
+    },
+    alignEnd: {
+      flexAfter: 0,
+    },
+    // vertical style overrides
+    isVertical: {
+      minHeight: 24,
+    },
+    hasChildren: {
+      contentPadding: 8,
```

Comment:
can probably use a global token for this, there should be examples of `globalTokens.size80` in other places